lib/cf/CloudControllerClient.js

```diff
@@ -227,19 +231,54 @@ class CloudControllerClient extends HttpClient {
   getSpaces(options) {
     return this.createSpaceStream(options).all();
   }
-  getOrg(space_guid, options) {
+
+  getSpace(space_guid, options) {
     return this.getResource(`/v2/spaces/${space_guid}`, options);
   }
+
   getOrgAndSpaceGuid(instance_guid) {
     return this.getServiceInstance(instance_guid)
-      .then(body => this.getOrg(body.entity.space_guid)
-        .then(org => ({
+      .then(body => this.getSpace(body.entity.space_guid)
+        .then(space => ({
           space_guid: body.entity.space_guid,
-          organization_guid: org.entity.organization_guid
+          organization_guid: space.entity.organization_guid
         }))
       );
   }
 
+  getServiceInstanceDetails(instance_guid) {
+    return this.getServiceInstance(instance_guid)
+      .then(instance => {
+        return Promise
+          .all([
+            this.getService(instance.entity.service_guid),
+            this.getServicePlan(instance.entity.service_plan_guid),
+            this.getSpace(instance.entity.space_guid)
+          ])
+          .spread((service, plan, space) => {
+            return _
+              .chain({})
+              .set('name', instance.entity.name)
+              .set('label', service.entity.label)
```

Comment:
change this attrib to `service_name` . We have never used `label` anywhere in code base, so it becomes confusing to understand what a label means just looking at job data.